api/wasm/README.md

```diff
@@ -0,0 +1,42 @@
+Envoy hooks:
```

Comment:
The `api/` tree is for xDS APIs, why is there Wasm docs and build files here?

Reply:
This is all going to be removed.  Pending https://github.com/envoyproxy/envoy-wasm/pull/613 which I will merge in here and ping this thread.